src/Utility/HttpStatus.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Helper class for dealing with HTTP status codes.
+ *
+ * @package Requests\Utilities
+ */
```

Comment:
```suggestion
/**
 * Requests for PHP, an HTTP library.
 *
 * @copyright 2012-2023 Requests Contributors
 * @license   https://github.com/WordPress/Requests/blob/stable/LICENSE ISC
 * @link      https://github.com/WordPress/Requests
 */
```

(For consistency with #834)